java/java.hints/src/org/netbeans/modules/java/hints/perf/SizeEqualsZero.java

```diff
@@ -51,11 +51,22 @@
     @BooleanOption(displayName = "#LBL_org.netbeans.modules.java.hints.perf.SizeEqualsZero.CHECK_NOT_EQUALS", tooltip = "#TP_org.netbeans.modules.java.hints.perf.SizeEqualsZero.CHECK_NOT_EQUALS", defaultValue=CHECK_NOT_EQUALS_DEFAULT)
     public static final String CHECK_NOT_EQUALS = "check.not.equals";
 
+    @BooleanOption(displayName = "#LBL_org.netbeans.modules.java.hints.perf.SizeEqualsZero.CHECK_GREATER_ZERO", tooltip = "#TP_org.netbeans.modules.java.hints.perf.SizeEqualsZero.CHECK_GREATER_ZERO", defaultValue=CHECK_NOT_EQUALS_DEFAULT)
+    public static final String CHECK_GREATER_ZERO = "check.greater_zero";
+
     @TriggerPatterns({
         @TriggerPattern(value="$subj.size() == 0", 
             constraints = @ConstraintVariableType(type = "java.util.Collection", variable = "$subj")),
         @TriggerPattern(value="$subj.size() == 0", 
             constraints = @ConstraintVariableType(type = "java.util.Map", variable = "$subj")),
+        @TriggerPattern(value="$subj.size() <= 0",
+            constraints = @ConstraintVariableType(type = "java.util.Collection", variable = "$subj")),
+        @TriggerPattern(value="$subj.size() <= 0",
+            constraints = @ConstraintVariableType(type = "java.util.Map", variable = "$subj")),
+        @TriggerPattern(value="$subj.size() < 1",
+            constraints = @ConstraintVariableType(type = "java.util.Collection", variable = "$subj")),
+        @TriggerPattern(value="$subj.size() < 1",
+            constraints = @ConstraintVariableType(type = "java.util.Map", variable = "$subj")),
```

Comment:
Shouldn't be here also the flipped conditions like `1 > $subj.size()` ?

Reply:
I will add them.

Reply:
Now added.